### PR TITLE
complete OKG launch

### PR DESCRIPTION
**Pull request by Buko-pie**
Reference issue https://github.com/mgguild/gamevault-frontend/issues/234
Capture1

Adjust render Claimtab to include TBA display if no vestingAddress in guildpad
Capture

### CODE REVIEW

**Review by Emman1598 — APPROVED** [review]

**Comment by kojiadrianojr** [comment]
alisin nalang ata si connec wallet jan sa claim OR ilagay sa AVAILABLE: instead na 0.0 ilagay nalang connect wallet

**Comment by swengr-janan** [comment]
> alisin nalang ata si connec wallet jan sa claim OR ilagay sa AVAILABLE: instead na 0.0 ilagay nalang connect wallet

What do you think kuys @medardm ?

File: src/views/GuildPad/Pad/Cards/Ido.tsx [hunk 1]
```diff
@@ -57,6 +58,15 @@ const CountDown: React.FC<{ status: string; round: string; start?: boolean; end?
   )
 }
 
+BigNumber.config({
```
Comment (medardm) [inline src/views/GuildPad/Pad/Cards/Ido.tsx:61]: naysu maray naisipan mo ites

Comment (medardm) [reply src/views/GuildPad/Pad/Cards/Ido.tsx:61]: watch out lang sa values na daks decimal places sa future

**Review by medardm — APPROVED** [review]
LGTM

**Pull request merged**